golang/lib/binding_constructors/binding_constructors.go

```diff
@@ -190,9 +186,9 @@ func NewRemoveServiceArgs(serviceId string, containerStopTimeoutSeconds uint64)
 //                                          Repartition
 // ==============================================================================================
 func NewRepartitionArgs(
-		partitionServices map[string]*kurtosis_core_rpc_api_bindings.PartitionServices,
-		partitionConnections map[string]*kurtosis_core_rpc_api_bindings.PartitionConnections,
-		defaultConnection *kurtosis_core_rpc_api_bindings.PartitionConnectionInfo) *kurtosis_core_rpc_api_bindings.RepartitionArgs {
+	partitionServices map[string]*kurtosis_core_rpc_api_bindings.PartitionServices,
```

Comment:
Same

Reply:
It's automatically generated when I use the command `Code > Reformat Code` in Go Land.
I'll move it as it was before.